compiler/one-cmds/tests/onnx_legalize_run_compare.py

```diff
@@ -0,0 +1,76 @@
+import onnxruntime as rt
+import onnx
+import sys
+import os
+import numpy as np
+
+
+def _generate_inputs(model):
+    inputs = {}
+    for input in model.graph.input:
+        # check if elem type is float32
+        assert (input.type.tensor_type.elem_type == 1)
+        input_shape = []
+        for dim in input.type.tensor_type.shape.dim:
+            input_shape += [dim.dim_value]
+        inputs[input.name] = np.random.random(input_shape).astype(np.float32)
+    return inputs
+
+
+def _run_model(model, inputs):
+    output_names = list(map(lambda output: output.name, model.graph.output))
+    session = rt.InferenceSession(model.SerializeToString())
+    outputs = session.run(output_names, inputs)
+    return outputs
+
+
+def _compare_resuts(ref_outputs, test_outputs, tolerance):
+    num_outputs = len(ref_outputs)
+    assert (len(test_outputs) == num_outputs)
+    for i in range(num_outputs):
+        if ref_outputs[i].shape != test_outputs[i].shape:
+            print("output {} shape mismatch".format(i))
```

Comment:
(optional) print `ref_outputs[i].shape` and `test_outputs[i].shape` itself may help debugging.